### PR TITLE
fix(renovate): move express-graphql and react-reconciler from minor to major update groups

### PR DESCRIPTION
**Pull request by hoobdeebla**
[express-graphql](/graphql/express-graphql) includes breaking changes in its minor releases. This is semver-compatible since the package is still below v1, but it breaks Renovate PRs that include this package. I'll investigate manually adapting to the changes, specifically [express-graphql#608](/graphql/express-graphql/pull/608) which seems to be the root of the errors generated after updating.

### CODE REVIEW

**Comment by wardpeet** [comment]
Shouldn't we include express-graphql to the list of majors for that package?

**Comment by hoobdeebla** [comment]
Made the change you suggested @wardpeet. Any new feedback?

**Comment by hoobdeebla** [comment]
I added react-reconciler to gatsby-recipes' majors list for the same reasons as listed above. With this new change applied to the existing updates in #27134, gatsby-recipes now successfully builds and passes all unit tests. gatsby was already passing all tests with the removal of updating express-graphql applied to #27092.

This should unblock both those PRs if my understanding of how Renovate works is correct. Any other changes I should make before this can get merged?

**Review by wardpeet — APPROVED** [review]

**Comment by wardpeet** [comment]
Thank you! 🙏 sorry for the delay!

**Pull request merged**
